cluster/src/main/java/com/hartwig/pipeline/PipelineMain.java

```diff
@@ -200,6 +200,7 @@ public PipelineState start(final Arguments arguments) {
                     apiStatusUpdateOrNot).run();
             VmExecutionLogSummary.ofFailedStages(storage, state);
             turquoise.publishComplete(state.status().toString());
+            turquoise.close();
```

Comment:
Now that the publisher is auto-closable it seems a great case for a try-with-resources statement. If you create the publisher in the `try` it will also be closed when an exception occurs.